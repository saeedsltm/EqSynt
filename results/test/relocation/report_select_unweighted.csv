OT,Lat,LatErr,Lon,LonErr,Dep,DepErr,NST,RMS,GAP
2020-01-01 00:01:40.000000,35.959,  4.70,51.784, 16.40, 24.1,  8.60,  9, 1.0,225
2020-01-01 00:03:18.400000,36.824, 13.10,50.854, 24.00, 22.1, 26.30,  5, 0.9,320
2020-01-01 00:05:00.400000,35.543,  7.50,51.003, 18.70, 25.7,999.90,  3, 0.6,313
2020-01-01 00:06:40.300000,35.370, 12.30,52.228, 19.30,  8.9, 18.00,  6, 0.7,189
2020-01-01 00:08:21.000000,35.683,125.30,51.607,108.40,  0.0,131.00,  6, 0.9,323
2020-01-01 00:09:59.700000,37.142,  7.80,52.262, 11.30, 15.0, 10.30,  7, 0.7,293
2020-01-01 00:11:40.400000,35.490, 11.80,52.079, 19.70,  6.9, 14.70,  4, 0.6,315
2020-01-01 00:13:19.500000,35.731,  3.70,52.304,  9.70,  3.4, 11.60,  8, 0.9,149
2020-01-01 00:14:59.900000,36.621,  6.80,51.915, 14.00, 23.9,  8.90,  6, 0.6,262
2020-01-01 00:16:41.100000,35.969,  4.70,51.690, 14.00, 16.3,  7.10, 10, 1.0,234
