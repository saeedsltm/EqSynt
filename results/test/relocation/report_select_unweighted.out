 Year Date HRMM  Sec Latitud Error Longitud Error Depth Error NST  RMS GAP
 2020  1 1 0001 40.0  35.959   4.7   51.784  16.4  24.1   8.6   9  1.0 225
 2020  1 1 0003 18.4  36.824  13.1   50.854  24.0  22.1  26.3   5  0.9 320
 2020  1 1 0005  0.4  35.543   7.5   51.003  18.7  25.7 999.9   3  0.6 313
 2020  1 1 0006 40.3  35.370  12.3   52.228  19.3   8.9  18.0   6  0.7 189
 2020  1 1 0008 21.0  35.683 125.3   51.607 108.4   0.0 131.0   6  0.9 323
 2020  1 1 0009 59.7  37.142   7.8   52.262  11.3  15.0  10.3   7  0.7 293
 2020  1 1 0011 40.4  35.490  11.8   52.079  19.7   6.9  14.7   4  0.6 315
 2020  1 1 0013 19.5  35.731   3.7   52.304   9.7   3.4  11.6   8  0.9 149
 2020  1 1 0014 59.9  36.621   6.8   51.915  14.0  23.9   8.9   6  0.6 262
 2020  1 1 0016 41.1  35.969   4.7   51.690  14.0  16.3   7.1  10  1.0 234
